fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBStreamAggregationTest.java

```diff
@@ -334,7 +334,8 @@ private void populateDB(final int numRecords) throws Exception {
     }
 
     @Nonnull
-    private RecordCursor<QueryResult> executePlan(final RecordQueryPlan plan, final int rowLimit, final byte[] continuation) {
+    private RecordCursor<QueryResult> executePlan(final RecordQueryPlan originalPlan, final int rowLimit, final byte[] continuation) {
+        final RecordQueryPlan plan = verifySerialization(originalPlan);
```

Comment:
This change alone is enough to surface the errors we were seeing in tests, but the other tests are nice to have and we want anyway (at least eventually).